src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/PersonalisationProvider.java

```diff
@@ -225,4 +224,40 @@ private static String readStringCaseField(final AsylumCase asylumCase, final Asy
         final Optional<String> optionalFieldValue = asylumCase.read(caseField, String.class);
         return optionalFieldValue.isPresent() && !optionalFieldValue.get().isEmpty() ? optionalFieldValue.get() : defaultIfNotPresent;
     }
+
+    public String getFtpaDecisionTemplateId(AsylumCase asylumCase,
```

Comment:
This method brakes concept of single responsibility principle for `PersonalisationProvider`,
you can have if else logic in both `*Personalisation` classes

Reply:
Will be refactored in later ticket - agreed.